README.md

```diff
@@ -1 +1,70 @@
-# math-magician
\ No newline at end of file
+# Getting Started with Create React App
+
```

Comment:
- [OPTIONAL] It will be good to write a descriptive README following the Microverse template.